src/log-agent/core/LogHandler.cs

```diff
@@ -254,61 +255,70 @@ private static void SetNgsaLogValues(NgsaLog nl)
-            }
-            else if (nl.Path.StartsWith("/api/genres", StringComparison.OrdinalIgnoreCase))
-            {
-                nl.Category = "PagedRead";
-                nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
-            }
+            //else if (nl.Path.StartsWith("/api/actors?", StringComparison.OrdinalIgnoreCase))
+            //{
+            //    nl.Category = "Ignore";
+            //    nl.Quartile = nl.Duration > 400 ? 4 : nl.Duration > 200 ? 3 : nl.Duration > 100 ? 2 : 1;
+            //}
+            //else if (nl.Path.StartsWith("/api/actors", StringComparison.OrdinalIgnoreCase))
+            //{
+            //    nl.Category = "Ignore";
+            //    nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
+            //}
+            //else if (nl.Path.StartsWith("/api/genres", StringComparison.OrdinalIgnoreCase))
+            //{
+            //    nl.Category = "Ignore";
+            //    nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
+            //}
             else if (nl.Path.StartsWith("/api/movies/", StringComparison.OrdinalIgnoreCase))
             {
                 nl.Category = "DirectRead";
                 nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
             }
             else if (nl.Path.StartsWith("/api/movies?", StringComparison.OrdinalIgnoreCase))
             {
-                nl.Category = "SearchMovies";
-                nl.Quartile = nl.Duration > 400 ? 4 : nl.Duration > 200 ? 3 : nl.Duration > 100 ? 2 : 1;
-            }
-            else if (nl.Path.StartsWith("/api/movies", StringComparison.OrdinalIgnoreCase))
-            {
-                nl.Category = "PagedRead";
-                nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
-            }
-            else if (nl.Path.StartsWith("/api/featured", StringComparison.OrdinalIgnoreCase))
-            {
-                nl.Category = "SearchMovies";
-                nl.Quartile = nl.Duration > 400 ? 4 : nl.Duration > 200 ? 3 : nl.Duration > 100 ? 2 : 1;
-            }
-            else if (nl.Path.StartsWith("/api", StringComparison.OrdinalIgnoreCase))
-            {
-                nl.Category = "PagedRead";
-                nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
-            }
-            else if (nl.Path.StartsWith("/healthz", StringComparison.OrdinalIgnoreCase))
-            {
-                nl.Category = "Healthz";
-                nl.Quartile = nl.Duration > 1600 ? 4 : nl.Duration > 800 ? 3 : nl.Duration > 400 ? 2 : 1;
-            }
-            else
-            {
-                nl.Category = "Static";
-                nl.Quartile = nl.Duration > 160 ? 4 : nl.Duration > 80 ? 3 : nl.Duration > 40 ? 2 : 1;
```

Comment:
I think you're effectively removing the log entry by making the ones we want to remove show up with an "Ignore" category.  If I got that right, I wonder if we could move the mappings for the category names to a configuration file of some sort that way code doesn't have to change in order to change the output?

Reply:
I added a todo to the code and will address in the next PR assuming we keep the current approach to logging.